src/services/email.service.spec.ts

```diff
@@ -0,0 +1,39 @@
+import nodemailer = require('nodemailer');
+import * as emailService from './email.service';
+
+describe('email service', () => {
+  test('send email failure missing credentials', async () => {
```

Comment:
@Whamo12 

Something you could do to reuse code would be to put the `userEmail, uuid & spy` all into a `beforeAll` 

Ex: 

```
let uuid;
let userEmail;
let spy;
BeforeAll(() => {
   // 
  // 
  //
}